### ISSUE TITLE
Contrast border is not visible for active activity bar item when activeBackground is set

### ISSUE THREAD

**Issue by usernamehw** [microsoft/vscode#109114]
1. Set both colors:
```js
"contrastActiveBorder": "#ff0000",
"activityBar.activeBackground": "#e4dbd6",
```
2. Activity bar contrast border is not visible.

Themes seems to be using translucent colors to workaround it. Is that `as-designed`?

**Comment by misolori** [comment]
Looks like a z-index issue here:

https://github.com/microsoft/vscode/blob/325cb4a67d5f3ef38123d7e5e9e4a3e8a2d5f000/src/vs/workbench/browser/parts/activitybar/activitybarActions.ts#L494-L501

Will fix in the next milestone since we are now in endgame for October.

**Comment by misolori** [comment]
You should be able to set both contrast border and active bg:

![image](https://user-images.githubusercontent.com/35271042/100128630-58a0a700-2e35-11eb-9f02-fa6c138a63b3.png)
